fuzzers/047a-hclk-idelayctrl-pips/README.md

```diff
@@ -0,0 +1,3 @@
+# HCLK_IOI interconnect fuzzer
+
+Solves pips located within the HCLK_IOI switch box.
```

Comment:
I guess some of the PR description should go into the READMEs.
Please say which PIPs exactly are being solved here.

Reply:
Done.